test_programs/execution_success/brillig_arrays/src/main.nr

```diff
@@ -9,7 +9,17 @@ fn main(x: [Field; 3]) {
     }
 }
 
-unconstrained fn read_array(x: [Field; 3]) {
+unconstrained fn read_array(
+    /// The signature internally will be represented as `(r, s)`
+    /// where `r` and `s` are fixed-sized big endian scalar values.
+    /// As the `secp256k1` has a 256-bit modulus, we have a 64 byte signature
+    /// while `r` and `s` will both be 32 bytes.
+    /// We expect `s` to be normalized. This means given an the curve's order,
+    /// `s` should be less than or equal to `order / 2`. 
+    /// This is done to prevent malleability. 
+    /// For more context regarding malleability you can reference BIP 0062.
```

Comment:
?

Reply:
this is an outdated version.  I was testing https://github.com/noir-lang/noir/issues/7711